src/tests/regress/data/sql/strings.sql

```diff
@@ -182,8 +182,8 @@ SELECT 'abcdefg' SIMILAR TO '_bcd\%' AS false;
 -- and an empty string to mean "no escape", which is also not per spec
 SELECT 'abcd\efg' SIMILAR TO '_bcd\%' ESCAPE '' AS true;
 -- these behaviors are per spec, though:
-SELECT 'abcdefg' SIMILAR TO '_bcd%' ESCAPE NULL AS null;
-SELECT 'abcdefg' SIMILAR TO '_bcd#%' ESCAPE '##' AS error;
+--@ SELECT 'abcdefg' SIMILAR TO '_bcd%' ESCAPE NULL AS null;
+--@ SELECT 'abcdefg' SIMILAR TO '_bcd#%' ESCAPE '##' AS error;
```

Comment:
Why are these tests disabled?

Reply:
So why?

Reply:
Forget to support NULL as escape char, the output is always NULL. I guess it's not important, I'll fix it later.

Reply:
The behavior is confusing. Escape ‘’ means no escape, and escape NULL results NULL output.